cli/cmd/root.go

```diff
@@ -148,14 +143,14 @@ func runVersionCheck(vChecker *version.VersionChecker, flags []string, cliConfig
 	}
 }
 
-// isInstallSubCommand checks if the subcommand is `install`
+// skipVersionCheck checks if the subcommand is `install` or `--oauth`
 // args here does not contain the main command
 // e.g., For `keptn -q install`, args would be just ['-q', 'install']
-func isInstallSubCommand(args []string) bool {
+func skipVersionCheck(args []string) bool {
```

Comment:
this is a bad name for function checking if the subcommand is install or --oauth

Reply:
... on the other hand, the sole purpose of this func is to determine whether we should skip the version check or not ;)